src/payment/checkout/payment-form/StripePaymentForm.jsx

```diff
@@ -133,8 +133,8 @@ function StripePaymentForm({
               city,
               country,
               line1: address,
-              line2: unit,
-              postal_code: postalCode,
+              line2: unit || '',
+              postal_code: postalCode || '',
```

Comment:
line2 and postal code are optional, but Stripe complains about it if we send `undefined`